src/fiskaltrust.Launcher/Commands/ConfigCommand.cs

```diff
@@ -0,0 +1,248 @@
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using fiskaltrust.Launcher.ProcessHost;
+using fiskaltrust.Launcher.Services;
+using Serilog;
+using ProtoBuf.Grpc.Server;
+using fiskaltrust.Launcher.Download;
+using fiskaltrust.Launcher.Extensions;
+using fiskaltrust.Launcher.Helpers;
+using fiskaltrust.Launcher.Common.Configuration;
+using fiskaltrust.Launcher.Common.Extensions;
+using fiskaltrust.Launcher.Configuration;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+
+namespace fiskaltrust.Launcher.Commands
+{
+
+    public class ConfigCommand : Command
+    {
+        public ConfigCommand() : base("config")
+        {
+            AddOption(new Option<string?>("--cipher-access-token"));
+
+            AddOption(new Option<string>("--launcher-configuration-file", getDefaultValue: () => "launcher.configuration.json"));
+            AddCommand(new ConfigSetCommand());
+            AddCommand(new ConfigGetCommand());
+        }
+    }
+
+    public class ConfigSetCommand : RunCommand
+    {
+        public ConfigSetCommand() : base("set", false)
+        {
+            AddOption(new Option<SemanticVersioning.Range?>("--launcher-version", parseArgument: arg => SemanticVersioning.Range.Parse(arg.Tokens.Single().Value)));
+        }
+    }
+
+    public class ConfigSetCommandHandler : ICommandHandler
+    {
+        public LauncherConfiguration ArgsLauncherConfiguration { get; set; } = null!;
+        public string LauncherConfigurationFile { get; set; } = null!;
+
+        public Guid? CipyerCashboxId { get; set; }
+        public string? CipyerAccessToken { get; set; }
```

Comment:
What does "Cipyer" mean?

Reply:
Should be Cipher 😅 as in used for encryption/decryption...
I was searching for a word that means both encryption and decryption at the same time...